pkg/network/ovn_kubernetes.go

```diff
@@ -435,43 +465,65 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 		fillKubeProxyDefaults(conf, nil)
 	}
 
-	// obtain the current IP family mode.
-	ipFamilyMode := names.IPFamilySingleStack
-	if len(conf.ServiceNetwork) == 2 {
-		ipFamilyMode = names.IPFamilyDualStack
-	}
-	// check if the IP family mode has changed and control the conversion process.
-	updateNode, updateMaster := shouldUpdateOVNKonIPFamilyChange(bootstrapResult.OVN, ipFamilyMode)
-	// annotate the daemonset and the daemonset template with the current IP family mode,
-	// this triggers a daemonset restart if there are changes.
-	err = setOVNObjectAnnotation(objs, names.NetworkIPFamilyModeAnnotation, ipFamilyMode)
-	if err != nil {
-		return nil, progressing, errors.Wrapf(err, "failed to set IP family %s annotation on daemonsets or statefulsets", ipFamilyMode)
+	zoneModeMigrationIsOngoing := isZoneModeMigrationAboutToStartOrOngoing(bootstrapResult.OVN, &targetZoneMode)
+	if zoneModeMigrationIsOngoing {
+		klog.Infof("migration to %s is ongoing or about to start", string(targetZoneMode.zoneMode))
```

Comment:
nit: klog should start with capitalized letters...not sure why CNO does not flag this in gofmt check